backends-common/postgres/src/main/java/org/apache/james/backends/postgres/PostgresTable.java

```diff
@@ -52,11 +53,58 @@ default FinalStage supportsRowLevelSecurity() {
         }
     }
 
+    public abstract static class AdditionalAlterQuery {
+        private String query;
+
+        public AdditionalAlterQuery(String query) {
+            this.query = query;
+        }
+
+        abstract boolean isApplied(boolean rowLevelSecurityEnabled);
```

Comment:
isApplied -> shouldBeApplied.

Also we could introduce an enum for `boolean rowLevelSecurityEnabled` and call it `RowLevelSecurity` with values `ENABLED` or `DISABLED`

Reply:
> Also we could introduce an enum for `boolean rowLevelSecurityEnabled` and call it `RowLevelSecurity` with values `ENABLED` or `DISABLED`

To change this, I have to refactor code in a lot places. I think rowLevelSecurityEnabled is enough to understand

Reply:
> To change this, I have to refactor code in a lot places. 

Not an issue to me.

https://www.teamten.com/lawrence/programming/prefer-enums-over-booleans.html

I'm surprised I need to argue to use strong types.

Reply:
fixed